### ISSUE TITLE
client 1.1.54046: "Failed to get default time zone information" if TZ points to file

### ISSUE THREAD

**Issue by Civil** [yandex/ClickHouse#196]
According to:
https://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html

```
The third format looks like this:

:characters
Each operating system interprets this format differently; in the GNU C Library, characters is the name of a file which describes the time zone.
```

In my environment, TZ=":/etc/localtime"
Which is valid symlink to zoneinfo. But clickhouse-client (stable) can't parse it.

"unset TZ" works as a workaround.

**Comment by alexey-milovidov** [comment]
Confirmed.

**Comment by ztlpn** [comment]
Solved in v1.1.54198.
